modular_citadel/code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1,11 +1,3 @@
-/datum/reagent/space_cleaner/reaction_obj(obj/O, reac_volume)
-	if(istype(O, /obj/effect/decal/cleanable) || istype(O, /obj/item/projectile/bullet/reusable/foam_dart) || istype(O, /obj/item/ammo_casing/caseless/foam_dart))
```

Comment:
This bit hasn't been properly demodularized. Space cleaner is supposed to clean up foam darts.